ui/app/rif/pages/addNewMulticryptoAddress/addNewMulticryptoAddress.js

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react'
```

Comment:
One thing about this class is that we should start making a common template. What this means is that we should have our components pretty similar, something like:
```
imports...

class ComponentClassName extends Component {
    state....
    propTypes....
    constructor....
    componentDidMount....(hooks)
    render...
}

mapStateToProps....
other functions....

module.exports ....
```
As you can see state and props types first, then constructor, then hooks (componentDidMount, etc) and at the end the render function, also the mapstatetoprops at the bottom of the file together with other functions and finally the module exports. We should have something like that, i think it could help us to improve the code and our workflow.

Reply:
You right, lets start using this convention